src/bidiMapper/domains/network/NetworkStorage.ts

```diff
@@ -66,7 +66,7 @@ export class NetworkStorage {
    */
   addIntercept(value: {
     urlPatterns: Network.UrlPattern[];
-    phases: Network.InterceptPhase[];
+    phases: [Network.InterceptPhase, ...Network.InterceptPhase[]];
```

Comment:
Does this mean an array with at least one element?

Reply:
Yes, the first one will always be present. See https://github.com/google/cddlconv/issues/16 for cddlconv issue.